docs/detections/rules-ui-manage.asciidoc

```diff
@@ -136,6 +136,8 @@ NOTE: The action frequency you select applies to all actions (both new and exist
 
 You can duplicate, enable, disable, and delete rules:
 
+NOTE: When duplicating a rule with exceptions, you can choose to duplicate the rule and its exceptions or duplicate the rule only. If you choose to duplicate the rule and its exception, copies of the exceptions are created and added to the duplicated rule's <<detections-ui-exceptions,default rule list>>.  
```

Comment:
I think it's worth explaining what happens when they choose to "duplicate the rule with its exceptions" and the rule contains a shared exception list. In this case, the shared exception list reference will be kept. In other words, the duplicated rule will also have a reference to the same shared list.